resources/views/cart/partials/product/description.blade.php

```diff
@@ -0,0 +1,16 @@
+<td class="flex max-md:w-1/2 md:table-cell">
+    <div class="flex flex-col items-start">
+        <a :href="item.url | url">
+            <div dusk="cart-item-name">@{{ item.name }}</div>
+            <div v-for="(optionValue, option) in item.options">
+                @{{ option }}: @{{ optionValue }}
+            </div>
+            <div v-for="option in cart?.items2?.find((item) => item.item_id == itemId).options.filter((option) => !['info_buyRequest', 'option_ids'].includes(option.code) && option.label)">
+                @{{ option.label }}: @{{ option.value.title || option.value }}
+            </div>
+        </a>
+        <button v-on:click="remove(item)" class="text-ct-inactive mt-1 text-xs hover:underline" :dusk="'item-delete-' + index">
```

Comment:
Nice 😄 . 

Only one thing, what if we want to change the remove button? 

- I want to remove the text and use a trash can icon
- I want to move the remove button to another place and I don't want it under the product name. 

Now we still have to override this whole file?

Reply:
If you want to move it around I don't really see any other way regardless, as you'd have to remove it from here and add it to wherever else you'd want.

I do think it's probably good---regardless of the extendability---to move this button into its own file because it's not really fitting for a component named "description". The component name is slightly misleading right now because this button is in there.